tostaylo/audio-effects
# #2 - First Impulse Response

# What Changed

- Added a ```wav``` file for a Fender Twin Reverb impulse response. 
- Modified gain function to use a constructor
- Modified .prettierignore to ignore files.

## How to test
Checkout this branch and then pull the latest. 
Make sure no current functionality is broken. The impulse response is a little quieter than the previous signal.  You can test by commenting out line 27 and 28 here in ```audio.js```. The sound without commented out code should be different than with commented code.  We will need to build a trigger which can easily add or remove nodes from the signal chain so these all or nothing effects will be easier to test. 

<img width="432" alt="image" src="https://user-images.githubusercontent.com/10001990/185813838-b8f5d9fe-6305-4386-b6c6-f32f57b93fa7.png">
